### ISSUE TITLE
dtool copy --resume results in IndexError

### ISSUE THREAD

**Issue by tjelvar-olsson** [jic-dtool/dtool-irods#7]
```
dtool copy --resume symlink:/my_ds yirods:/zone/colllection
Copying dataset
Traceback (most recent call last):
 File "/usr/local/bin/dtool", line 11, in <module>
   sys.exit(dtool())
 File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
   return self.main(*args, **kwargs)
 File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
   rv = self.invoke(ctx)
 File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
   return _process_result(sub_ctx.command.invoke(sub_ctx))
 File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
   return ctx.invoke(self.callback, **ctx.params)
 File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
   return callback(*args, **kwargs)
 File "/usr/local/lib/python2.7/dist-packages/dtool_create/dataset.py", line 369, in copy
   progressbar=progressbar
 File "/usr/local/lib/python2.7/dist-packages/dtoolcore/__init__.py", line 207, in copy_resume
   _copy_content(dataset, proto_dataset, progressbar)
 File "/usr/local/lib/python2.7/dist-packages/dtoolcore/__init__.py", line 137, in _copy_content
   dest_sizes = get_dest_sizes(dest_proto_dataset)
 File "/usr/local/lib/python2.7/dist-packages/dtoolcore/__init__.py", line 131, in get_dest_sizes
   for handle in dest_proto_dataset._storage_broker.iter_item_handles():
 File "/usr/local/lib/python2.7/dist-packages/dtool_irods/storagebroker.py", line 533, in iter_item_handles
   relpath = self._get_metadata_with_cache(abspath, "handle")
 File "/usr/local/lib/python2.7/dist-packages/dtool_irods/storagebroker.py", line 273, in _get_metadata_with_cache
   value = value_line.split(":")[1]
IndexError: list index out of range
```

**Comment by tjelvar-olsson** [comment]
This defect arises when a file is put into iRODS, but the connection to iRODS breaks before the subsequent task of adding the "handle" metadata to the file has been performed.

**Comment by tjelvar-olsson** [comment]
Fixed in 50287d9a1dfef0cde12aaff3d043c595034e08ab
